Makefile

```diff
@@ -30,14 +79,17 @@ version:
 	@echo $(VERSION)
 
 ./bin/node-problem-detector: $(PKG_SOURCES)
-	GOOS=linux go build -o bin/node-problem-detector \
+	CGO_ENABLED=$(CGO_ENABLED) GOOS=linux go build -o bin/node-problem-detector \
 	     -ldflags '-w -extldflags "-static" -X $(PKG)/pkg/version.version=$(VERSION)' \
```

Comment:
@ankon Good catch. This is introduced during last rebase. As is mentioned in https://github.com/kubernetes/node-problem-detector/pull/54#pullrequestreview-14090197, we should not try to static link when journald is enabled.